test/TestApps/ToastNotificationsDemoApp/main.cpp

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include "pch.h"
+#include <wil/win32_helpers.h>
+#include <iostream>
+#include <winrt/Windows.ApplicationModel.Background.h>
+#include "WindowsAppRuntime.Test.AppModel.h"
+#include <MddBootstrap.h>
+
+using namespace winrt::Microsoft::Windows::AppLifecycle;
```

Comment:
can we create namespace winrt? and use winrt all over the file instead of big extensions?